index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Slider</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="slider">
+
+        <div class="slider__block">
+            <div class="slider__line">
+                <img src="images/1.jpg" alt="">
+                <img src="images/2.jpg" alt="">
```

Comment:
Як БЕМ, то може дамо класи зображенням?
```suggestion
<img src="images/2.jpg" alt="Віддзеркалення рук дорослого та дитини в дзеркалі, яке тримає дівчина. Дівчина лежить на асфальті пофарбованому в жовтий та блакитний" class="slider__image" />
```